lib/core/backup_service.dart

```diff
@@ -148,14 +157,29 @@ class BackupService {
     await _importPreferencesDump();
   }
 
+  // checked with .endsWith - so this should be the last part of the filename
+  static const ignoreFiles = [
+    "flutter_assets/kernel_blob.bin",
+    "flutter_assets/vm_snapshot_data",
+    "flutter_assets/isolate_snapshot_data",
+    ".lock",
+  ];
```

Comment:
out of curiosity, why do we ignore them?

Reply:
as for "flutter_assets/kernel_blob.bin", "flutter_assets/vm_snapshot_data", "flutter_assets/isolate_snapshot_data" if we backup/restore these files the app will crash in runtime when on debug build
and .lock files are not required - so I just included them while also adding the list - so we have even less files in backup.